### PR TITLE
Task #79214 view profile

### PR DESCRIPTION
**Pull request by IamRedsan**
## Related Tickets
- [Task #79214](https://edu-redmine.sun-asterisk.vn/issues/79214)

## HOW
- View profile

## Evidence (Screenshot or Video)
image

### CODE REVIEW

File: app/static/css/profile.css
Comment (ferus311) [inline app/static/css/profile.css:1]: end file

**Review by ferus311 — APPROVED** [review]

**Comment by IamRedsan** [comment]
ready

**Comment by IamRedsan** [comment]
ready

**Comment by IamRedsan** [comment]
ready

**Comment by IamRedsan** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]

**Pull request merged**